lib/publiccloud/instance.pm

```diff
@@ -255,19 +260,26 @@ sub upload_log {
     assert_script_run("test -d '$tmpdir' && rm -rf '$tmpdir'");
 }
 
-=head2 wait_for_guestregister
+=head2 wait_for_guestregister_chk
 
-    wait_for_guestregister([timeout => 300]);
+    wait_for_guestregister_chk([timeout => 300]);
 
 Run command C<systemctl is-active guestregister> on the instance in a loop and
 wait till guestregister is ready. If guestregister finish with state failed,
 a soft-failure will be recorded.
 If guestregister will not finish within C<timeout> seconds, job dies.
 In case of BYOS images we checking that service is inactive and quit
 Returns the time needed to wait for the guestregister to complete.
+C<wait_for_guestregister_chk> is called inside C<create_instance()>, enabled by C<check_guestregister>
 =cut
 
 sub wait_for_guestregister {
+    my ($self, %args) = @_;
+    my $out = $self->run_ssh_command(cmd => 'sudo systemctl is-active guestregister', proceed_on_failure => 1, quiet => 1);
+    return;
+}
+
+sub wait_for_guestregister_chk {
```

Comment:
Can you documment this subroutine? I don't know what `_chk` means.

Reply:
See previous comment  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16577#discussion_r1136998017

Reply:
done